src/resources/nfts/types.ts

```diff
@@ -0,0 +1,55 @@
+import { Network } from '@/helpers/networkTypes';
+import { AssetContract, AssetType } from '@/entities';
+import { UniqueTokenType } from '@/utils/uniqueTokens';
+
+type NFTMarketplace = {
+  collectionId: string | null;
+  collectionUrl: string | null;
+  floorPrice: number | null;
+  id: string | null;
+  name: string | null;
+  nftUrl: string | null;
+};
+
+type NFTCollection = {
+  description: string | null;
+  discord: string | null;
+  externalUrl: string | null;
+  imageUrl: string | null;
+  name: string | null;
+  simplehashSpamScore: number | null;
+  twitter: string | null;
+};
+
+export type NFTTrait = {
+  displayType: string | null;
+  traitType: string;
+  value: string | number;
+};
+
+export type NFT = {
```

Comment:
For others reviewing: these types are unused atm, they just represent the shape we're moving towards.